### ISSUE TITLE
exposed CSS code

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-studio#1271]
### Steps to Reproduce

See https://ocw.mit.edu/courses/6-004-computation-structures-spring-2017/pages/c1/c1s1/ 

I'm not sure how widespread this in. If we can identify the issue, then it would be good to do a search through studio data. 

If it's just a couple of pages, we can ask some to remove it manually through studio.

**Comment by MAbdurrehman12** [comment]
@pdpinch It looks like this raw CSS is in some legacy courses, I was able to do a quick search and filter out some courses.
Not sure if this list is considered long, or medium but I think this is something that will not occur in new courses so maybe we can do it manually.
 
If we decide to remove it manually then here are the links to studio (of pages) containing this raw CSS: 

Note: 
studio RC link can be replaced with prod.
It also contains some resources link: for images, there is this raw CSS in the image caption.

```
Format
Course Name
Studio site link

6-006-introduction-to-algorithms-fall-2011

https://ocw-studio-rc.odl.mit.edu/sites/6-006-introduction-to-algorithms-fall-2011/type/page/edit/2e91de4c-23a9-f323-866d-265121e71f07/ 
https://ocw-studio-rc.odl.mit.edu/sites/6-006-introduction-to-algorithms-fall-2011/type/page/edit/3e392c68-f217-c957-d2c8-70f39c63c084/


21a-550j-dv-lab-documenting-science-through-video-and-new-media-fall-2012

https://ocw-studio-rc.odl.mit.edu/sites/21a-550j-dv-lab-documenting-science-through-video-and-new-media-fall-2012/type/page/edit/d61a4e55-ca51-6fb4-fe89-4ab0e52d9a13/


6-004-computation-structures-spring-2017
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/24191c34-3d57-2641-409b-fd54b3d9b2dc/?q=1.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/76b26f4e-38da-1486-221a-ccb9b343c762/?q=10.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/2db1eadc-5344-f2de-9462-f7932dabec44/?q=11.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/29840bcc-41c4-bfab-a071-981783ab0a1b/?q=12.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/7fba5e1d-2fa4-056c-150f-90508b97f937/?q=13.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/76ea0269-3b06-ecaa-e0bb-d8ca4be4bb7e/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/61c8d864-831f-2145-e0a9-922732a385ec/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/435fa3f2-0748-7b97-ace6-65391f3bff5f/?q=17.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/5e79a00f-7377-2e87-4bc8-390515e63a0d/?q=18.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/afb5e6a8-476a-7f76-4e0a-4be0d22685e4/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/f36c8cab-16b9-624b-b4fe-c82e9ddf0f73/?q=2.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/866cbcd7-fe65-4979-53fa-90baa0c3f6da/?q=20.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/9a439586-e23c-50b5-9bf2-cfbc1289a41e/?q=21.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/d77171d7-bbaa-8eb5-d974-6a2895701253/?q=3.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/b39a5b15-c1c8-c348-7bc5-29f6a8d2e699/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/ba996719-fd35-c417-5993-5d31177dfcb9/?q=5.1+Annotated+Slides
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/1ce0a254-f5ec-8356-abea-088ff937a0ac/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/a97ebc62-7536-6091-9846-90b005674a5d/
https://ocw-studio-rc.odl.mit.edu/sites/6-004-computation-structures-spring-2017/type/page/edit/c8a85f9c-129e-5b36-f9c5-2a3365574124/?q=8.1+Annotated+Slides


18-065-matrix-methods-in-data-analysis-signal-processing-and-machine-learning-spring-2018
https://ocw-studio-rc.odl.mit.edu/sites/18-065-matrix-methods-in-data-analysis-signal-processing-and-machine-learning-spring-2018/type/resource/edit/7d689bdb-8da9-848c-569f-85e93ce9b9d4/
https://ocw-studio-rc.odl.mit.edu/sites/18-650-statistics-for-applications-fall-2016/type/resource/edit/74db4119-31d8-54f5-d226-78b891acf562/?q=18-650f16-th.jpg
https://ocw-studio-rc.odl.mit.edu/sites/18-650-statistics-for-applications-fall-2016/type/resource/edit/de1506da-6268-0127-2f3f-35bb8412e638/?q=18-650f16.jpg
https://ocw-studio-rc.odl.mit.edu/sites/6-849-geometric-folding-algorithms-linkages-origami-polyhedra-fall-2012/type/page/edit/a370594e-3650-963e-bb62-70f5dc3b68ed/

```
  
If you need more info like studio admin links or UUIDs, let me know.

**Comment by pdpinch** [comment]
@MAbdurrehman12 can you just go ahead and fix these pages? You can do it through Studio, or some other way if you like.

**Comment by MAbdurrehman12** [comment]
> can you just go ahead and fix these pages? You can do it through Studio, or some other way if you like.

Sure

**Comment by MAbdurrehman12** [comment]
These pages have been fixed on both, RC and PROD.
However, some pages had missing descriptions (page description as seen by search engines) which is a required field and we can not save the content without it so I also added them (added what seemed the most suitable to me) but they can be updated (if required).

Pages of this course `6-004-computation-structures-spring-2017` mentioned [above](https://github.com/mitodl/ocw-studio/issues/1271#issuecomment-1125757499) had missing descriptions.
